src/main/java/se/sundsvall/parkingpermit/util/TimerUtil.java

```diff
@@ -12,7 +12,7 @@ public final class TimerUtil {
 	private TimerUtil() {}
 
 	public static Date getControlMessageTime(Decision decision, String controlMessageDelay) {
-		final var decisionCreated = Optional.ofNullable(decision.getCreated()).orElse(OffsetDateTime.now());
+		final var decisionCreated = decision != null ? Optional.ofNullable(decision.getCreated()).orElse(OffsetDateTime.now()) : OffsetDateTime.now();
```

Comment:
Instead of mixing ternary operator and Optional.ofNullable: maybe ha a nullcheck for both? 

```
final var decisionCreated = OffsetDateTime.now();
if(nonNull(decision) && nonNull(decision.getCreated())) {
   decisionCreated = decision.getCreated();
}
```

Reply:
Approved with comment